packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -52,6 +52,17 @@ class AndroidDevice extends Device {
   final String modelID;
   final String deviceCodeName;
 
+  @override
+  String toString() => name;
+  /*
+    "\n"
+    "name: $name,\n"
+    "id: $id,\n"
+    "productID: $productID,\n"
+    "modelID: $modelID,\n"
+    "deviceCodeName: $deviceCodeName\n";
+  */
+
```

Comment:
Please remove this, if this is something you added for debugging purposes. Let's try our best to not modify the existing flutter code.

Also, in general, it is not a good practice to check in commented out code, unless you have a strong reason for doing so.